appinventor/appengine/src/com/google/appinventor/client/Ode.java

```diff
@@ -404,12 +402,12 @@ public void run() {
           // a second press while the new project wizard was starting (aka we "debounce"
           // the button). When the person switches to the projects list view again (here)
           // we re-enable it.
-          projectToolbar.enableStartButton();
-          projectToolbar.setProjectTabButtonsVisible(true);
-          projectToolbar.setTrashTabButtonsVisible(false);
+          bindProjectToolbar.enableStartButton();
+          bindProjectToolbar.setProjectTabButtonsVisible(true);
+          bindProjectToolbar.setTrashTabButtonsVisible(false);
         }
       };
-    if (designToolbar.getCurrentView() != DesignToolbar.View.BLOCKS) {
+    if (bindDesignToolbar.getCurrentView() != DesignToolbar.View.BLOCKS) {
```

Comment:
Was there a particular need to rename the fields here? Otherwise, the names could be changed back reducing the number of lines changed from master.

Reply:
> Was there a particular need to rename the fields here? Otherwise, the names could be changed back reducing the number of lines changed from master.

I actually changed the variable names back in my Phase 2 branch. I changed them to help me sort some issues where multiple instances were being created of singleton classes and was reluctant to change back while I was debugging. But the reasons why I was keeping a separate variable name are all resolved now.